django/views/generic/dates.py

```diff
@@ -300,7 +300,11 @@ def _make_single_date_lookup(self, date):
 
 
 class BaseDateListView(MultipleObjectMixin, DateMixin, View):
-    """Abstract base class for date-based views displaying a list of objects."""
+    """
+    Base class for date-based views displaying a list of objects.
+
+    Using this base class requires subclassing to provide a response mixin.
```

Comment:
I think we can tighten this sentence up, since from context it’s clear we’re already talking about “using this base class”:

```suggestion
    Requires subclassing to provide a response mixin.
```